### ISSUE TITLE
Multiple PKCS#7 messages for testing unwrapping

### ISSUE THREAD

**Issue by Javex** [Javex/libscep#27]
We need test messages for all possible cases we want to unwrap:
- PKCSReq
- CertRep (with SUCCESS, FAILURE and PENDING) for each possible type (PKCSReq, GetCertInitial, GetCert, GetCRL)
- GetCertInitial
- GetCert
- GetCRL

Since these are a lot of messages, we need some kind of mechanism to remove them from `test_message.c`. One possibility would be to have them on the filesystem but then the compiled test files would need the accomanying files. Another idea would be to put them in header files but that makes an ugly header file (or multiples).

However, we could combine those ideas: We put them as files on the filesystem and during compilation have a script that reads those files and puts them in a header file. Then our test file just needs to include this file. Downside is a possible dependency for development (e.g. perl, python) but we can live with that.

**Comment by Javex** [comment]
Here is how to generate messages using `openca-scep`: https://github.com/openxpki/openxpki/tree/908559d9e4db4758b0ac15271c1beaffd6a5c586/core/server/OpenXPKI/Crypto/Tool/SCEP/Command

**Comment by gknocke** [comment]
All testfiles which can be generated by openca-scep can also be generated by ourselves. We do not need to create them via different tools anymore. 
I think the dependency if fine.

**Comment by Javex** [comment]
Correct, but we don't have an easy command line interface yet and hacking that stuff together just to generate these requests takes unnecessary time. I fully support replacing the test messages with our own once our implementation of the CLI clients is finished.

**Comment by gknocke** [comment]
Hm.. please take a look at "own_certrep_pending" in test_message.c.
I created the requests by my own using our certrep implementation. When checking the unwrap functionality, I first generate this request by my own. This is the difference to certrep_pending wich comes from the outside. So it seems to work: Why not using our API to generate the requests? Its basically just one additional line of code.

**Comment by Javex** [comment]
I removed myself as an assignee, maybe I'll pick this up later, but I've written enough tests for now.
